.ci/azure-pipelines-build.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   - task: NodeTool@0
     displayName: 'Install Node'
     inputs:
-      versionSpec: '16.x'
+      versionSpec: '18.18.x'
```

Comment:
It may be preferred to leave this as `18.x` and change `package.json` to target `18.17`.

Reply:
:+1: I would prefer sticking with `18.x`